lib/elements/elements.css

```diff
@@ -196,34 +196,108 @@
 }
 
 .toast {
-  padding: 0.5em 1em;
-  position: absolute;
   right: 16px;
   top: 0;
   min-width: 12em;
   max-width: 40em;
+  padding: 0.5em 1em;
 
-  opacity: 0;
-  transition: 400ms;
   -webkit-user-select: none;
 }
 
 .toast.showing {
-  opacity: 1;
   top: 72px;
 }
 
+.toast.dialog {
+  z-index: 18;
+}
+
+.glass-pane {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 0.40;
+  background-color: #222;
+  z-index: 15;
+  transition: 200ms;
+}
+
 .dialog {
-  background: rgba(255, 255, 255, 0.06);
+  position: absolute;
+  padding: 10px;
+  background-color: #444;
   border: 1px solid;
   border-color: rgba(0, 0, 0, 0.5);
   border-radius: 4px;
   box-shadow: rgba(255, 255, 255, 0.1) 0 0 0, rgba(0, 0, 0, 0.4) 0 1px 7px 0;
+  
+  opacity: 0;
+  transition: 400ms;
   z-index: 20;
 }
 
+.dialog.showing {
+  opacity: 1;
+}
+
+.dialog .title {
+  font-size: 16pt;
+  white-space: nowrap;
+}
```

Comment:
I had a bottom border on the title area for a while. I wasn't 100% sold on it. I may play around with the style you have here, and try and call back to the 4px border below the page title.